apps/webapp/src/state/send.ts

```diff
@@ -96,26 +91,31 @@ export const createSendSlice = (): SliceCreator<SendSlice> => (set, get) => {
   };
 };
 
-const getExponent = (asset: TempAsset): number => {
-  const match = asset.denomUnits.find(u => u.denom === asset.display);
-  return match?.exponent ?? 0;
-};
+const getExponent = (asset: AssetBalance): number => asset.denom.exponent;
```

Comment:
If this is the function, we don't need it and can put `asset.denom.exponent` below